### ISSUE TITLE
Not enough Haskamos

### ISSUE THREAD

**Issue by AbrahamGreenman** [schorrm/ypp#1]
I feel like there aren't enough haskamos for this open source project. I l'maseh would only want to use it if it's chasuv. Right now don't feel chasuv enough

**Comment by michaelbdavidson7** [comment]
Nu, I would say there are takeh too many. Look through the shprach yourself and taineh your own pshatim!

**Comment by schorrm** [comment]
I’d be delighted to have more haskamos. It’d be a great kovid

**Comment by davidGrocerKey** [comment]
those asking this must not be familiar with my rebbi the Great Rav Assaf Bednarsh,  his shining haskama should be enough for any thinking ben torah thumbs up!
